qiskit/aqua/components/__init__.py

```diff
@@ -24,7 +24,7 @@
 :mod:`~qiskit.aqua.components.variational_forms` that can be chosen from according the nature of
 the problem. Some optimizers use gradients, others have alternative techniques to finding a
 minimum. Variational forms include heuristic ansatzes such as
-:class:`~qiskit.aqua.components.variational_forms.RYRZ` and types designed for specific problems
+:class:`~qiskit.circuit.library.EfficientSU2` and types designed for specific problems
```

Comment:
I do not know whether this will resolve. I guess it depends on how the overall docs from the elements are built - if they are built as one then it will work, but I am thinking its independent builds that are brought together by the web page for the documentation.

Reply:
#1185 changed the docs as it seems everything is being built together now. So the above should resolve I think - we'll see.